op-deployer/pkg/deployer/upgrade/v2_0_0/upgrade_test.go

```diff
@@ -21,6 +21,7 @@ import (
 )
 
 func TestUpgradeOPChainInput_OpChainConfigs(t *testing.T) {
+	t.Skip("skipping for upgrade 15")
```

Comment:
why are these skipped? they should have to be

Reply:
Uses local artifacts and the interface has changed - we need to fix this by replacing the local artifacts with tagged artifacts. Can we do this as part of a fast-follow PR?

Reply:
sure, happy to do it as part of a fast-follow. should just be a few lines of code to do the tagged artifacts.